docs/Administration.rst

```diff
@@ -3061,6 +3064,136 @@ this entry is updated:
 -  The number of added users is increased if some users were added to the bound group by an administrator
 
 
+.. _AdminAppCenter:
+
+============================
+Managing Application Center
+============================
+
+As a platform administrator, you can manage the Application Center by:
+
+-  Defining the number of allowed favorite applications
+-  Defining mandatory applications that appear in every favorite applications drawer/list
+-  Defining default icon for added applications
+-  Adding/Editing applications
+
+To access the administration menu for Application Center, you need to access to the left navigation menu through
+the hamburger icon -> Administration -> OTHER -> Applications
```

Comment:
The menu label changed. Please update the path

Reply:
ok

Reply:
the navigation location has been changed, the label OTHER is still the same, I'll keep it as is until it is synchronized with crowdin